src/test/java/core/basesyntax/dao/CustomFileReaderTest.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax.dao;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class CustomFileReaderTest {
+    private static final String REPORT_TO_READ = "src/main/resources/reportToRead.csv";
+    private static final String EMPTY_REPORT = "src/main/resources/emptyReport.csv";
+    private static CustomFileReader customFileReader;
+
+    @BeforeAll
+    static void beforeAll() {
+        customFileReader = new CustomFileReaderImpl();
+    }
+
+    @Test
+    void readOk() {
```

Comment:
Make method more informative. Use two or three parts 
```suggestion
    void read_validReport_ok() {
```